microk8s-resources/wrappers/run-kubelite-with-args

```diff
@@ -218,3 +228,27 @@ fi
 
 declare -a args="($(cat $SNAP_DATA/args/$app))"
 exec "$SNAP/$app" "${args[@]}"
+
```

Comment:
The argument cleanup must be done before we start the kubelite daemon, ie before the `exec $SNAP/$app` command above.